### ISSUE TITLE
add --network_train_unet_only option

### ISSUE THREAD

**Issue by knishika62** [cocktailpeanut/fluxgym#19]
I don't know why, but adding --network_train_unet_only makes the faces more similar.

**Comment by marhensa** [comment]
> I don't know why, but adding --network_train_unet_only makes the faces more similar.

where do you add it?

**Comment by knishika62** [comment]
> where do you add it?

Advanced options

**Comment by marhensa** [comment]
> > where do you add it?
> 
> Advanced options (default OFF)

like add it on the run arguments?

`python app.py --network_train_unet_only` or where to put it on Advanced Options?

because I don't see where to put it on Advanced Options GUI

![image](https://github.com/user-attachments/assets/bc9a7020-64c9-4ed1-87b9-73e4b09482a7)

**Comment by knishika62** [comment]
> because I don't see where to put it on Advanced Options GUI

Put it in the advanced options GUI. Is it the lowest?  like a this.

--network_train_unet_only
[pulldown] 
off(default) | on

If it is on, add this option to cli.

accelerate launch   --mixed_precision bf16   --num_cpu_threads_per_process 1   sd-scripts/flux_train_network.py   --pretrained_model_name_or_path "D:\fluxgym\models\unet\flux1-dev.sft"   --clip_l "D:\fluxgym\models\clip\clip_l.safetensors"   --t5xxl "D:\fluxgym\models\clip\t5xxl_fp16.safetensors"   --ae "D:\fluxgym\models\vae\ae.sft"   --cache_latents_to_disk   --save_model_as safetensors   --sdpa --persistent_data_loader_workers   --max_data_loader_n_workers 2   --seed 42   --gradient_checkpointing   --mixed_precision bf16   --save_precision bf16   --network_module networks.lora_flux   --network_dim 4   --optimizer_type adamw8bit   --learning_rate 8e-4   --cache_text_encoder_outputs   --cache_text_encoder_outputs_to_disk   --fp8_base   --highvram   --max_train_epochs 15   --save_every_n_epochs 5   --dataset_config "D:\fluxgym\dataset.toml"   --output_dir "D:\fluxgym\outputs"   --output_name test-8e4-dm4   --timestep_sampling shift   --discrete_flow_shift 3.1582   --model_prediction_type raw   --guidance_scale 1   --loss_type l2 **--network_train_unet_only**
